packages/types/src/codec/U8a.spec.ts

```diff
@@ -49,6 +49,6 @@ describe('U8a', () => {
   });
 
   it('implements subarray correctly', () => {
-    expect(u8a.subarray(1, 3).toString()).toEqual('0x0203');
+    expect(u8a.subarray(1, 3).toString()).toEqual('2,3');
```

Comment:
Actually here, would be nicer to remove the toString, and toEqual(Uint8Array.from())